web/app/frontend/src/components/form/TextInput.tsx

```diff
@@ -4,7 +4,7 @@ import {Text, TextInput as TextInputRNP} from "react-native-paper";
 import {darkTheme} from "../../constants/theme";
 import {InputProps} from "./FormControl";
 
-const TextInput: React.FC<InputProps> = (
+const TextInput = (
```

Comment:
why this change?

Reply:
We're not sure, but somehow tsserver was complaining.

Also: https://react-typescript-cheatsheet.netlify.app/docs/basic/getting-started/function_components